gstd/gstd.c

```diff
@@ -233,7 +240,9 @@ main (gint argc, gchar * argv[])
   }
 
   if (nodaemon) {
-    print_header (quiet);
+    if (!quiet) {
```

Comment:
Please do something with the `print_header`, I don't feel it's okay we have 3 different places where we print it.

Reply:
Fixed